html/BrokerNode.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+require_once("requests/IriWrapper.php");
+require_once("requests/IriData.php");
+require_once("requests/NodeMessenger.php");
+require_once("iota-php-port/PrepareTransfers.php");
+
+class BrokerNode
+{
+    public static $chunksToAttach = null;
+    public static $chunksToVerify = null;
+
+    public static $IriWrapper = null;
```

Comment:
Can we set this to `new IriWrapper()` here? Lazy instantiation doesn't seem necessary.

Reply:
My IDE gives me this error if I try to do that:  "expression is not allowed as field default value."  I think it's because it's static.  

We could change the variables and methods in this file to not be static if we wanted, but I was trying to not instantiate BrokerNode, IriWrapper, and NodeMessenger over and over, since we'll be calling them thousands of times.  The approach I've taken here might not be the best way to do it and we could revisit it.  

I am going to make the other change though.

Reply:
Ok, let's revisit later. I'm not a PHP expert, so I don't know what the best solution would be